### PR TITLE
covid 19 metrics webapp

### PR DESCRIPTION
**Pull request by Enekwestanley**
Hello reviewer, For this PR, these are the highlight implemented from the project requirement: Configure the react app and connect to our redux store. Fetch data and update our redux store. Render our UI from our store. Write actions and reducers for fetching the data from the API. Render some conditional components and also used the filtering method Setup react testing libraries and all the component tests are passing Deployed the project No linter errors Followed the correct git-flow Thank you 👍🏽

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,189 @@
+<a name="readme-top"></a>
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (fobadara) [inline README.md]: [REQUIRED] Kindly fix or remove this logo because it is not displaying properly.

File: README.md [hunk 2]
```diff
@@ -1,73 +1,189 @@
...
 
+**This web app was created using React and Redux. It is used to track the Covid-19 metrics update for the Top 20 African countries in view.**
+## 🛠 Built With <a name="built-with"></a>
+- React
+- HTML5
+- Linters
+- GitFlow
+- Redux
+  
+<img src="src/assests/mock.png" alt="logo"   height="auto" />
```
Comment (fobadara) [inline README.md]: [REQUIRED] Kindly fix this image too. It is not dispaying

File: README.md [hunk 3]
```diff
@@ -1,73 +1,189 @@
...
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Deployment <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://covid-19-webb-app.netlify.app/)
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
```
Comment (fobadara) [inline README.md]: [OPTIONAL] This is confusing. I think you should have just named the video link something like video presentation or any other descriptive name.

File: README.md [hunk 4]
```diff
@@ -1,73 +1,189 @@
...
 
 Give a ⭐️ if you like this project!
 
-## Acknowledgments
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
```
Comment (fobadara) [inline README.md]: [REQUIRED] Kindly acknowledge Nelson Sakwa as described in the project requirements

File: src/components/_test_/__snapshots__/Details.test.js.snap [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+  "findByTitle": [Function],
+  "getAllByAltText": [Function],
+  "getAllByDisplayValue": [Function],
+  "getAllByLabelText": [Function],
+  "getAllByPlaceholderText": [Function],
+  "getAllByRole": [Function],
+  "getAllByTestId": [Function],
+  "getAllByText": [Function],
+  "getAllByTitle": [Function],
+  "getByAltText": [Function],
```
Comment (fobadara) [inline src/components/_test_/__snapshots__/Details.test.js.snap:209]: [REQUIRED] I can't seem to figure out the purpose of these texts. Kindly remove them if they are not serving any purpose.

**Review by fobadara — CHANGES_REQUESTED** [review]
# REQUIRES CHANGES ♻️ 
Hi @Enekwestanley, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing
✅ Good first attempt
✅ Project looks professional

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/_test_/__snapshots__/Details.test.js.snap (hunk 5 above)
Comment (Enekwestanley) [reply src/components/_test_/__snapshots__/Details.test.js.snap:209]: They are automatically generated  when I run test.

**Review by AnselemOdims — APPROVED** [review]
### Hi  @Enekwestanley 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**